tests/coverage.sh

```diff
@@ -24,19 +27,30 @@ ADD examples /var/lib/githooks/examples
 RUN git config --global user.email "githook@test.com" && \
     git config --global user.name "Githook Tests"
 
-ADD tests/exec-steps.sh tests/${STEPS_TO_RUN} tests/replace-inline-content.py /var/lib/tests/
+ADD tests/exec-steps.sh tests/${STEPS_TO_RUN} /var/lib/tests/
 
 # Some fixup below:
 # We overwrite the download to use the current install.sh in all scripts
 RUN \\
 # Make sure we're using Bash for kcov
-    find /var/lib -name '*.sh' -exec sed -i 's|#!/bin/sh|#!/bin/bash|g' {} \\; && \\
-    find /var/lib -name '*.sh' -exec sed -i 's|sh /|bash /|g' {} \\; && \\
-    find /var/lib -name '*.sh' -exec sed -i 's|sh "|bash "|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|#!/bin/sh|#!/bin/bash|g' {} \\; && \\
+    # at the beginnig of line
+    find /var/lib -name '*.sh' -exec sed -i -E 's|^( *)sh /|\1bash /|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|^( *)sh "|\1bash "|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|^( *)sh ~/|\1bash /home/coverage/|g' {} \\; && \\
+    # in between line
+    find /var/lib -name '*.sh' -exec sed -i -E 's|( +)sh /|\1bash /|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|( +)sh "|\1bash "|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|( +)sh ~/|\1bash /home/coverage/|g' {} \\; && \\
+    # git hooks alias
+    find /var/lib -name '*.sh' -exec sed -i 's|"!sh |"!bash |g' {} \\; && \\
 # Revert changed shell script filenames
     find /var/lib -name '*.sh' -exec sed -E -i "s|/var/lib/githooks/([a-z-]+)\\.bash|/var/lib/githooks/\\1.sh|g" {} \\; && \\
-# Replace the inline content with loading the source file
-    python /var/lib/tests/replace-inline-content.py /var/lib/githooks && \\
+# Change any \`git hooks\` invocation to the shell script for better code coverage
+    find /var/lib/tests/ -name '*.sh' -exec sed -i -E 's|([^"])git +hooks|\1bash /home/coverage/.githooks/release/cli.sh|g' {} \\; && \\
+    find /var/lib/tests/ -name '*.sh' -exec sed -i -E 's|^( +)git +hooks|\1bash /home/coverage/.githooks/release/cli.sh|g' {} \\; && \\
+# Change multiline echos to line-wise echos for kcov
+    sed -i -E '/echo "$/,/^"/{  s/echo "/echo ""/ ; s/^"$/echo ""/  ;  /^\s*echo/! { s/(.*)/echo "\1"/ } }' /var/lib/githooks/cli.sh && \\
```

Comment:
Couldn't we just leave them as is and accept some drip in % coverage? How bad it is?